tests/pr_test.go

```diff
@@ -59,12 +59,18 @@ func TestRunBasicExample(t *testing.T) {
 	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
 		Testing:      t,
 		TerraformDir: basicExampleTerraformDir,
-		Prefix:       "hpcs",
-		TerraformVars: map[string]interface{}{
-			"existing_kms_instance_guid": permanentResources["hpcs_south"],
-		},
+		Prefix:       "kms-key-basic",
+		Region:       "us-south",
 	})
 
+	terraformVars := map[string]interface{}{
+		"prefix":                     options.Prefix,
+		"region":                     options.Region,
```

Comment:
actually the basic example doesnt have a region input var, so please remove this